zerver/webhooks/uptimerobot/doc.md

```diff
@@ -0,0 +1,36 @@
+Receive Zulip notifications for your Uptime Robot monitors!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+    Make sure to specify the topic in the URL above. Otherwise, the
+    default topic `General` will be used.
+
+1. Go to your **My Settings** on Uptime Robot, and select **Add Alert Contact**
```

Comment:
"your **My Settings**" sounds a bit unwieldy, perhaps simply "**My Settings**" would be better? And I think the ``.`` got moved to the new line?